gateway/src/main/resources/messages/login_de.properties

```
@@ -0,0 +1,11 @@
+### Translation file
+login_message_title = Bitte identifizieren Sie sich
+username = Nutzername
+password = Password
+login = Anmelden
+register = Anmeldung
+forget_password = Passwort vergessen ?
+identity_provider_title = Melden Sie sich bei einem Identitätsanbieter an
+expired_password = Ihr Passwort ist abgelaufen
+expired_password_link = und sollte geändert werden
+invalid_credentials = ungültiger Benutzername oder Passwort
```

Just some translation suggestions:
```suggestion
login_message_title = Bitte melden Sie sich an
username = Nutzername
password = Passwort
login = Anmelden
register = Registrieren
forget_password = Passwort vergessen?
identity_provider_title = Melden Sie sich bei einem Identitätsanbieter an
expired_password = Ihr Passwort ist abgelaufen
expired_password_link = und sollte geändert werden
invalid_credentials = Ungültiger Benutzername oder Passwort
```